sqs/src/main/scala/akka/stream/alpakka/sqs/SqsAckFlowStage.scala

```diff
@@ -100,7 +96,7 @@ private[sqs] final class SqsAckFlowStage(queueUrl: String, sqsClient: AmazonSQSA
             val (message, action) = grab(in)
             val responsePromise = Promise[AckResult]
             action match {
```

Comment:
I like the approach to delete or change the message visibility. From my perspective it would be great to have a DoNothing option to simply do nothing and wait for the message to reappear.